app/controllers/application_controller.rb

```diff
@@ -135,7 +137,7 @@ def append_info_to_payload(payload)
   def info_for_paper_trail
     {
       user_id: warden&.user&.id,
-      session_id: request.env['rack.session.record']&.session_id,
+      session_id: session&.id&.to_s,
```

Comment:
are we sure this should be an integer?

Reply:
It's a UUID. I just copied the `to_s` since Theron had that on the `GraphqlController` (when we set this field on activity logs) 👀

Reply:
ok, weird that it's in `.id` but I'm good if the value is actually a UUID.